src/search/include/search/search_tree.h

```diff
@@ -279,23 +333,21 @@ std::ostream &operator<<(std::ostream &os, const search::NodeLabel &node_label);
  */
 template <typename Location, typename ActionType>
 void
-print_to_ostream(std::ostream &                                                   os,
+print_to_ostream(std::ostream &                                      os,
                  const search::SearchTreeNode<Location, ActionType> &node,
-                 bool         print_children = false,
-                 unsigned int indent         = 0)
+                 bool                                                print_children = false,
+                 unsigned int                                        indent         = 0)
 {
-	for (unsigned int i = 0; i < indent; i++) {
-		os << "  ";
-	}
 	os << "(" << indent << ") -> { ";
-	// TODO This should be a separate operator
-	for (const auto &action : node.incoming_actions) {
-		os << "(" << action.first << ", " << action.second << ") ";
-	}
 	os << "} -> " << node.words << ": " << node.state << " " << node.label;
-	if (print_children) {
+	if (false && print_children) {
```

Comment:
That is a weird way of commenting out blocks of code =D

Reply:
Good catch :smile: